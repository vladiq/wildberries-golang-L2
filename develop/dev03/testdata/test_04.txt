234432 00032 234023
234420 3242304
23423 2034323 24234234
2432423 23
234 111
0 22