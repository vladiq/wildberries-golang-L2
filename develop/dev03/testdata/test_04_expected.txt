0 22
2432423 23
234432 00032 234023
234 111
23423 2034323 24234234
234420 3242304